crates/pathfinder/src/rpc.rs

```diff
@@ -642,6 +642,7 @@ mod tests {
         ];
 
         let transaction_receipts = vec![
+            // FIXME: declare transactions don't have events, so this is invalid test data!
```

Comment:
This should be fixed before merging this I think.

I mean this was true even before this PR, but now we try to explicitly test the `getTransactionReceipt` RPC result which exposes the receipts.